utils/netns/netns.go

```diff
@@ -59,3 +61,32 @@ func IsVethExists(vethHostName string) (bool, error) {
 	}
 	return false, nil
 }
+
+// GetLinkLocalAddr Construct the IPv6 link local address of interfaceName
+// from the interface MAC address.
+func GetLinkLocalAddr(interfaceName string) net.IP {
+	link, e := netlink.LinkByName(interfaceName)
+	if e != nil {
+		log.Errorf("GetLinkLocalAddr: %v", e)
+		return nil
+	}
+	mac := link.Attrs().HardwareAddr
```

Comment:
This approach is a little surprising. I had expected you to just fetch the IP address rather than constructing it from the MAC. What was the rational for doing it this way?